bismuthcore/bismuthcore/legacy/peershandler.py

```diff
@@ -451,9 +453,9 @@ def consensus_remove(self, peer_ip):
             self.app_log.info(f"Consensus opinion list: {self.peer_opinion_dict}")
             self.app_log.info(f"Will remove {peer_ip} from consensus pool {self.peer_opinion_dict}")
             self.peer_opinion_dict.pop(peer_ip)
-        except:
+        except Exception as e:
+            self.debug(f"Error removing {peer_ip} from consensus pool: {e}")
```

Comment:
dup log (plus typo likely, self.debug instead of self.app_log.debug)

Reply:
See comments regarding `legacy/`